commands/active.go

```diff
@@ -21,3 +27,38 @@ func cmdActive(c *cli.Context) {
 		fmt.Println(host.Name)
 	}
 }
+
+func getActiveHost(store persist.Store) (*host.Host, error) {
```

Comment:
I moved this into `active.go` from `ls.go` because it wasn't referenced in `ls.go` at all...